miscellaneous/build_templates/analysis.yml

```diff
@@ -3,6 +3,7 @@ parameters:
   skipPolaris: false # For samples that have no code Polaris is capable of scanning
   useRequirementsTxt: false # Tell the task to use Python requirements.txt for BlackDuck
   projectPath: '' # (Optional) Tell the task to use a specific path to the project
+  snippetMatching: None
```

Comment:
did you say you changed the default to `SNIPPET_MATCH`? did you push the change?

Reply:
I put it in a different branch. I have updated the correct branch now.